app/javascript/components/Components/Header.js

```diff
@@ -1,45 +1,36 @@
-import React, { Component } from "react"
+import React from "react"
 import { Nav, NavItem } from 'reactstrap'
 
-export default class Header extends Component {
-    render(){
-        const {
-            logged_in,
-            sign_in_route,
-            sign_up_route,
-            sign_out_route
-        } = this.props
-        return(
-            <div id="header">
-
-                <h3 className="title">PageLender</h3>
-
-                <Nav>
-                    
-                    <NavItem>
-                        <a href="/">Home</a>
-                    </NavItem>
-                   
-                    <NavItem>
-                        <a href="/library">Library</a>
-                    </NavItem>
-                   
-                    <NavItem>
-                        <a href="/borrow">Borrow</a>
-                    </NavItem>
-                   
-                    <NavItem>
-                        <a href="/lend">Lend</a>
-                    </NavItem>
-
-                    { logged_in && 
-                        <NavItem>
-                            <a href={sign_out_route}>Sign Out</a>
-                        </NavItem>
-                    }
-
-                </Nav>
-            </div>
-        )
-    }
-}
\ No newline at end of file
+const Header = (props) => (
+
+    <div id="header">
+
+        <h3 className="title">PageLender</h3>
+
+        <Nav>
+
+            <NavItem>
+                <a href="/">Home</a>
+            </NavItem>
+
+            <NavItem>
+                <a href="/library">Library</a>
+            </NavItem>
+
+            <NavItem>
+                <a href="/borrow">Borrow</a>
+            </NavItem>
+
+            <NavItem>
+                <a href="/lend">Lend</a>
+            </NavItem>
+
+            <NavItem>
```

Comment:
Did you want to keep your `logged_in` check?

Example:

`{ props.logged_in && <NavItem><a href={props.sign_out_route}>Sign Out</a></NavItem>}`

Reply:
I believe it's not necessary. We have our devise login page as our root when not authenticated so I believe they will have to be logged in to see this page anyway.